src/commonTest/kotlin/fr/acinq/lightning/channel/states/SpliceTestsCommon.kt

```diff
@@ -192,22 +222,27 @@ class SpliceTestsCommon : LightningTestSuite() {
     fun `reject splice_ack`() {
         val cmd = createSpliceOutRequest(25_000.sat)
         val (alice, bob) = reachNormal()
-        val (_, actionsAlice1) = alice.process(cmd)
-        val (bob1, actionsBob1) = bob.process(ChannelCommand.MessageReceived(actionsAlice1.hasOutgoingMessage<SpliceInit>()))
+        val (nodes, _, _) = addHtlc(15_000_000.msat, alice, bob)
+        val (alice0, bob0) = crossSign(nodes.first, nodes.second)
+        val (_, bobQuiescent, spliceInit) = reachQuiescent(cmd, alice0, bob0)
```

Comment:
Naming this state `bobQuiescent` isn't consistent, we end up mixing naming for alice and bob (you never use `aliceQuiescent`). It's tedious but more consistent to just increase the index of each state.